addons/sourcemod/configs/vsh/vsh.cfg

```diff
@@ -546,7 +546,8 @@
 
 		"772"	//Baby Face Blaster
 		{
-			"desp"			"Baby Face Blaster: {negative}Boost is reset on stun"
+			"desp"			"Baby Face Blaster: {negative}Boost takes 400 damage to fully charge and is reset on stun"
```

Comment:
iirc last time i tested, Soda Popper actually takes 380 damage to fill up meter (valve why)